ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -95,4 +116,16 @@ pub(crate) struct BottomUpRelayerArgs {
         help = "The max parallelism for submitting checkpoints"
     )]
     pub max_parallelism: usize,
+    #[arg(
+        long,
+        default_value = "true",
+        help = "Flag to turn on/off Prometheus metrics reporting."
+    )]
+    pub enable_metrics: bool,
+    #[arg(
+        long,
+        default_value = "0.0.0.0:9185",
+        help = "Local port for exporting Prometheus metrics."
+    )]
+    pub metric_export_bind_address: String,
```

Comment:
```suggestion
    pub metrics_listen_addr: String,
```

I call these `listen_addr` in fendermint. Also note that you called it `metric` here and `metrics` in the other setting.